lib/school.rb

```diff
@@ -0,0 +1,37 @@
+class School
+  attr_reader :start_time, :hours_in_school_day, :student_names
+
+  def initialize(start, hours)
+    @start_time = start
+    @hours_in_school_day = hours
+    @student_names = []
+    @end_time = (@start_time.to_i + @hours_in_school_day)
```

Comment:
I like the thought here but I think calculating `end_time` should be its own method and not an attribute.  This is simply due to the face what if the start time changes? We would need to recalculate this attribute somehow